src/components/ProgressBar/index.tsx

```diff
@@ -4,10 +4,16 @@ import ProgressBar from './ProgressBar';
 
 interface WrapperTypes {
   progress: number;
+  progressTitle: string;
```

Comment:
Because of this extra prop, we're getting this complaint on the `src/components/ProgressBar/ProgressBar.test.tsx`

But that should go away if you choose not to use this prop anymore, like I commented it'd be preferable on the `index.tsx`.

<img width="774" alt="image" src="https://user-images.githubusercontent.com/43002117/233673166-95edbb5d-ae91-4784-bc0f-3c391028484a.png">